tmb-user-mgt/tmb-userdata/src/main/java/de/interseroh/tmb/user/client/UserInformationService.java

```diff
@@ -0,0 +1,84 @@
+package de.interseroh.tmb.user.client;
+
+import com.google.gwt.core.client.Callback;
+import com.google.gwt.user.client.Cookies;
+import org.fusesource.restygwt.client.MethodCallback;
+import org.gwtbootstrap3.client.ui.base.ComplexWidget;
+
+import java.util.Collection;
+import java.util.logging.Logger;
+
+/**
+ * Abstracting the user related login and  user information services in order to enable a development-mode mocking
+ *
+ */
+public interface UserInformationService {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1214)

Reply:
geht net!